Makefile

```diff
@@ -34,4 +34,8 @@ GOFLAGS=""
 test:
 	@(./test/shell-pipeline-tests/openvino-bike-rentals/pipelines-test-openvino-bike-rentals.sh)
 	@(./test/shell-pipeline-tests/tensorflow-housing/pipelines-test-tensorflow-housing.sh)
+
+	@(oc create namespace test-namespace && oc project test-namespace)
```

Comment:
Can you separate this into a separate target for running `go test`? Until we can refactor the and unify the setup for the `shell` and `go test` framework, we'll need to have separate targets to prevent CI/CD runs from failing when both are not aligned